src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -869,20 +861,79 @@ public void canNormalizePatternStopTimes() throws IOException, SQLException, Inv
         TripDTO createdTrip = mapper.readValue(createTripOutput, TripDTO.class);
         // Update pattern stop with new travel time.
         JdbcTableWriter patternUpdater = createTestTableWriter(Table.PATTERNS);
-        int updatedTravelTime = 3600; // one hour
-        pattern.pattern_stops[1].default_travel_time = updatedTravelTime;
+        pattern.pattern_stops[updatedStopSequence].default_travel_time = updatedTravelTime;
         String updatedPatternOutput = patternUpdater.update(pattern.id, mapper.writeValueAsString(pattern), true);
         LOG.info("Updated pattern output: {}", updatedPatternOutput);
         // Normalize stop times.
         JdbcTableWriter updateTripWriter = createTestTableWriter(tripsTable);
-        updateTripWriter.normalizeStopTimesForPattern(pattern.id, 0);
+        updateTripWriter.normalizeStopTimesForPattern(pattern.id, 0, interpolateStopTimes);
+
+        return createdTrip.trip_id;
+    }
+
+    /**
+     * Checks that {@link JdbcTableWriter#normalizeStopTimesForPattern(int, int, boolean)} can interpolate stop times between timepoints.
+     */
+    @Test
+    public void canInterpolatePatternStopTimes() throws IOException, SQLException, InvalidNamespaceException {
```

Comment:
I updated this in https://github.com/conveyal/gtfs-lib/pull/399/commits/36f3043f11e761ea3a738f61174ec4ab5e363c31

Why do we use the public access modifier for the other tests but not here?

Reply:
@philip-cline the public modifier is not needed on any test. This has been highlighted since we started using SonarLint (do you have it installed btw?). We could create a PR and remove all, but for now any new methods should be created without it.